spec/lib/spotlight/dor/indexer_spec.rb

```diff
@@ -498,4 +498,87 @@
     end # each
   end # add_folder_name
   # rubocop:enable Metrics/LineLength
+
+  # rubocop:disable Metrics/LineLength
+  describe '#add_object_full_text' do
+    let(:expected_text) do
+      'SOME full text string that is returned from the server'
+    end
+    let(:full_file_path) do
+      'https://stacks.stanford.edu/file/oo000oo0000/oo000oo0000.txt'
+    end
+    let(:public_xml_with_feigenbaum_full_text) do
```

Comment:
Not to chime-in unnecessarily but I just wanted to clear something up about `let` vs `let!`.   The primary difference between the bang version (`let!`) is that it the value is evaluated immediately instead of lazily (i.e. `let` is evaluated when the variable is invoked where `let!` is evaluated immediately when set).

The availability of the value in the `let`/`let!` is more about the scope in which it is defined.

https://www.relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let